docs/howto/docker.rst

```diff
@@ -201,6 +201,36 @@ process the ``Dockerfile`` and handle the special syntax. Whereas the
 only generated once and then re-used by all subsequent pushes until the image
 is modified.
 
+Layering images on top of each other
+....................................
+
+The `parent` value on a docker-image task can refer to another image's `name`.
+The parent image can then be referenced from the dependent image's Dockerfile
+as `$DOCKER_IMAGE_PARENT`.
```

Comment:
An example showing both the task definitions + a Dockerfile snippet would be useful here.

Reply:
Might also be nice to link to `build-decision` and maybe add a warning about `taskgraph build-image` not being able to process this.

Reply:
Updated, thanks for the suggestions.

A rendered version is at https://taskcluster-taskgraph--631.org.readthedocs.build/en/631/howto/docker.html FWIW.